src/utils/api.ts

```diff
@@ -0,0 +1 @@
+export const sanitizeId = (id: string) => (id ? id.replace(/\//g, '_') : '');
```

Comment:
replaces `/` with `_` to prevent invalid paths

Reply:
does backend replaces it backwards? Or how does it work?

Reply:
it doesn't - I'm not sure if someone should be providing an ID that is invalid - we should probably throw an error if so and ensure we document that the id is expected to have no _ in it?

sanitizing the ID like we do in the main app is maybe more of an app-specific thing than a sdk thing?

Reply:
The BE will generate a uuid and return that in the response. The client should just override its id value with whatever is returned from the server. 

This is because sometimes the spinamp frontend has ids like `local/xyz` and that breaks the request routing. I realise this is a spinamp FE specific issue but including `/` in the path will still break our routing for any client so I think it's better to filter it out in the SDK

Reply:
right - but shouldn't we filter it out with an error rather than sanitizing it?